src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests.vb

```diff
@@ -16,24 +16,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
                              <file name="c.vb">
                                  <![CDATA[
 Public Class B2
-    Public Shared Operator +(x As B2, y As B2) As B2 
+    Public Shared Operator +(x As B2, y As B2) As B2
```

Comment:
It looks like there are no meaningful changes in this file, consider reverting to the previous state.